ArduPlane/AP_Arming.cpp

```diff
@@ -83,12 +83,15 @@ bool AP_Arming_Plane::pre_arm_checks(bool display_failure)
         ret = false;
     }
 
-    if (plane.channel_throttle->get_reverse() && 
-        Plane::ThrFailsafe(plane.g.throttle_fs_enabled.get()) != Plane::ThrFailsafe::Disabled &&
-        plane.g.throttle_fs_value < 
-        plane.channel_throttle->get_radio_max()) {
-        check_failed(display_failure, "Invalid THR_FS_VALUE for rev throttle");
-        ret = false;
+    if (Plane::ThrFailsafe(plane.g.throttle_fs_enabled.get()) != Plane::ThrFailsafe::Disabled) {
+        if (plane.channel_throttle->get_reverse() && plane.g.throttle_fs_value < plane.channel_throttle->get_radio_max()) {
+            check_failed(display_failure, "Invalid THR_FS_VALUE for rev throttle");
+            ret = false;
+        }
+        if (!(rc().has_had_rc_receiver() || rc().has_had_rc_override())) {
```

Comment:
this prevents arming in AUTO with THR_FAILSAFE=2, which is what tends to be used for BVLOS operations where R/C may not be connected on initial takeoff

Reply:
So change to enabled rather than not disabled?